CHANGELOG.md

```diff
@@ -45,6 +45,9 @@ This release is built on top of [OpenTelemetry .NET](https://github.com/open-tel
   `OTEL_DOTNET_AUTO_DISABLED_INSTRUMENTATIONS`&#8594;`OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS`
   `OTEL_DOTNET_AUTO_INSTRUMENTATION_PLUGINS`&#8594;`OTEL_DOTNET_AUTO_TRACES_PLUGINS`
   `OTEL_DOTNET_AUTO_ADDITIONAL_SOURCES`&#8594;`OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES`
+
+- `OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS` default value is changed to
```

Comment:
`OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` 😉 

We should change the title (and description) of this PR that at this moment it is only about "tracing instrumentations"